sdk/identity/identity/TROUBLESHOOTING.md

```diff
@@ -453,27 +457,39 @@ Once you've verified Azure PowerShell is using the correct account, validate tha
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AADSTS900023: Specified tenant identifier '975f013f-7f24-47e8-a7d3-abc4752bf34' is neither a valid DNS name, nor a valid external domain.                                                                                                                                                                                                                                | The tenant id passed to the credential is invalid                                                                                                                                 | Verify the tenant id is valid. If the service connection was configured via a user-assigned managed identity, the tenant will be the one in which managed identity was registered. If the service connection is configured via a Service Principal, the tenant should be the one in which the Service Principal is registered.                                                                             |
+| No service connection found with identifier <GUID>                                                                                                                                                                                                                                                                                                                       | The service connection id provided is incorrect.                                                                                                                                  | Verify the serviceConnectionId provided. This parameter refers to the `resourceId` of the Azure Service Connection. It can also be found in the queryString of the respective Service Connection's configuration page in Azure Devops.                                                                                                                                                                     |
```

Comment:
nit: 
```suggestion
| No service connection found with identifier <GUID>                                                                                                                                                                                                                                                                                                                       | The service connection id provided is incorrect.                                                                                                                                  | Verify the serviceConnectionId provided. This parameter refers to the `resourceId` of the Azure Service Connection. It can also be found in the query string of the respective Service Connection's configuration page in Azure DevOps. More information about service connections can be found [here](https://learn.microsoft.com/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) |
```